qutip/simdiag.py

```diff
@@ -78,75 +89,32 @@ def simdiag(ops, evals=True):
             if (A * B - B * A).norm() / (A * B).norm() > tol:
                 raise TypeError('Matricies must commute.')
 
-    A = ops[0]
-    eigvals, eigvecs = la.eig(A.full())
-    zipped = list(zip(-eigvals, range(len(eigvals))))
-    zipped.sort()
-    ds, perm = zip(*zipped)
-    ds = -np.real(np.array(ds))
-    perm = np.array(perm)
-    eigvecs_array = np.array(
-        [np.zeros((A.shape[0], 1), dtype=complex) for k in range(A.shape[0])])
-
-    for kk in range(len(perm)):  # matrix with sorted eigvecs in columns
-        eigvecs_array[kk][:, 0] = eigvecs[:, perm[kk]]
+    # TODO: rewrite using Data object
```

Comment:
Is this TODO for now? If not, do we need to add it to one of the TODO lists in the PR?

Reply:
Not for now, it's more a TODO until the full release.